esp_wrover_kit/include/bsp/display.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+/**
+ * @file
+ * @brief BSP LCD
+ *
+ * This file offers API for basic LCD control.
+ * It is useful for users who want to use the LCD without a Graphical Library or without the default LVGL library.
+ *
+ * For standard LCD initialization, you can call all-in-one function bsp_display_start().
+ */
+
+#pragma once
+#include "esp_lcd_types.h"
+
+/**
+ * @brief BSP display configuration structure
+ */
+typedef union {
+    struct {
+        int num;               /*!< Index of SPI peripheral to use */
+        int clk_hz;            /*!< SPI clock in Hz */
+        int max_transfer_size; /*!< SPI max transfer size in bytes */
+    } spi;
+} bsp_display_config_t;
```

Comment:
Why we need this structure? The BSP knows these values. All these values should be used from BSP header file.